api-dummy/config/packages/prod/monolog.yaml

```diff
@@ -0,0 +1,16 @@
+monolog:
```

Comment:
There is an assumption about which are the standard environments supported: `dev`, `prod`, what else?

Reply:
Depends on the infrastructure. `ci`/`test`; we could end up with `end2end` putting all the distributed Libero projects together. Maybe a `demo` that we host?

Reply:
The name of environments are good, we should only take care:
- not to put in assumptions about hostnames which we usually don't
- leave open the possibility to add new environments without forking the code (e.g. extending the Docker image and adding files)